app/frontend/actions.ts

```diff
@@ -1110,6 +1110,15 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     })
   }
 
+  const getErrorBannerContent = (state) => {
+    const errorBannerContent = ADALITE_CONFIG.ADALITE_ERROR_BANNER_CONTENT
+    const showErrorBanner = errorBannerContent !== '' && true
+    setState({
+      errorBannerContent,
+      showStakingBanner: showErrorBanner ? false : state.showStakingBanner,
```

Comment:
I think it would be cleaner for the staking banner showing logic to check if error banner is being shown rather than the error banner overriding this. This way the logic to show/hide the staking banner would be kept in a single place and it would be less likely to lead to confusion in the future